test/potluck_test.rb

```diff
@@ -0,0 +1,26 @@
+require "pry"
+require "minitest/autorun"
+require "minitest/pride"
+require "./lib/dish"
+require "./lib/potluck"
+
+class PotluckTest < Minitest::Test
+
+  def test_if_it_exists
+    potluck = Potluck.new
+    assert_instance_of Potluck, potluck
+  end
+
+  def test_if_it_adds_dishes
+    potluck = Potluck.new
+    couscous_salad = Dish.new("Couscous Salad", :appetizer)
+    assert_equal "Couscous Salad", potluck.add_dish("Couscous Salad")
```

Comment:
We don't want to pass the string "Couscous Salad" to the add_dish method, we want to pass the Dish object `couscous_salad`